iceoryx_posh/test/moduletests/test_popo_publisher_port.cpp

```diff
@@ -202,26 +206,29 @@ TEST_F(PublisherPort_test,
 
 TEST_F(PublisherPort_test, allocatingAChunk)
 {
-    auto maybeChunkHeader = m_sutNoOfferOnCreateUserSide.tryAllocateChunk(10u);
+    auto maybeChunkHeader = m_sutNoOfferOnCreateUserSide.tryAllocateChunk(
```

Comment:
In theory it could be possible that the custom header arguments are just ignored inside of `tryAllocate`. Therefore I would add a test here that verifies that `tryAllocate` with a custom header size != `iox::CHUNK_NO_CUSTOM_HEADER_SIZE` works and provides a chunk with the correct size/alignment